packages/bot-web-ui/src/analytics/utils.ts

```diff
@@ -44,3 +44,16 @@ export const getTradeParameterData = ({ form_values }: TFormStrategy) => {
         initial_stake: hasStoredText(stored_texts?.stake) ? stored_texts?.stake : stake,
     };
 };
+
+export const getStrategyType = (block_string: string) => {
+    try {
+        const xmlDoc = new DOMParser().parseFromString(block_string, 'application/xml');
+        if (xmlDoc.getElementsByTagName('xml').length) {
+            const root = xmlDoc.documentElement;
+            const isDbotValue = root.getAttribute('is_dbot');
+            return isDbotValue === 'true' ? 'new' : 'old';
+        }
```

Comment:
```suggestion
        }
        return 'old';
```
Seems like the logic is always default to 'old'. Doing this would make it so that this function always return a string and remove the need to assert as string in the lines below

Reply:
Done 👍